eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -87,10 +93,44 @@ stages:
       useMonoRuntime: false              # 🚫
       withPreviousSDK: false             # 🚫
 
-  - ${{ if ne(variables['Build.Reason'], 'PullRequest') }}:
+  - template: ../jobs/vmr-build.yml
+    parameters:
+      # Changing the build name requires updating the referenced name in the source-build-sdk-diff-tests.yml pipeline
+      buildName: CentOSStream8_Online_PreviousSourceBuiltSdk
+      isBuiltFromVmr: ${{ parameters.isBuiltFromVmr }}
+      vmrBranch: ${{ variables.VmrBranch }}
+      architecture: x64
+      pool:
+        name: ${{ variables.defaultPoolName }}
+        demands: ${{ variables.defaultPoolDemands }}
+      container: ${{ parameters.centOSStream8Container }}
+      buildFromArchive: false            # 🚫
+      enablePoison: false                # 🚫
+      excludeOmniSharpTests: true        # ✅
+      runOnline: true                    # ✅
+      useMonoRuntime: false              # 🚫
+      withPreviousSDK: true              # ✅
 
-    # CI - Stage 1 x64 legs ------------------------------------
+  - template: ../jobs/vmr-build.yml
+    parameters:
+      # Changing the build name requires updating the referenced name in the source-build-sdk-diff-tests.yml pipeline
+      buildName: Alpine317_Offline_PreviousSourceBuiltSdk
```

Comment:
Have you uploaded the previous artifacts for Alpine yet?  We will have to update the release automation to account for this change so that they are captured.

Reply:
Are you implying that I would need to update the release pipeline to publish the Alpine sdk?

Reply:
The release pipeline doesn't consume artifacts from Alpine: https://github.com/dotnet/source-build/blob/d3272e646a47287bef1f1dd4b21e86f54dc863c4/eng/templates/stages/mirror.yml#L74-L78

Reply:
> Are you implying that I would need to update the release pipeline to publish the Alpine sdk?

Yes.

Reply:
Addressing in [this issue](https://github.com/dotnet/source-build/issues/3635).